### PR TITLE
Add Mudirac

### PR DESCRIPTION
**Pull request by subindev-d**
- Added mudirac 
- Minor fixes for Cif2Cell and pymuonsuite tools

### CODE REVIEW

**Review by patrick-austin — CHANGES_REQUESTED** [review]
The functional changes are good, and have already been reviewed when merging one by one into `dev`.
For all the other tools, where we've just modified the label, strictly speaking we have [updated the wrapper without changing the underlying version](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions), so I think we should bump `@WRAPPER_VERSION@`. I'd have left suggested changes for this but can't as it's outside the vicinity of the actual change on each file, but it should be simple enough.

File: muspinsim/muspinsim.xml [hunk 1]
```diff
@@ -51,8 +51,8 @@
         <collection name="muspinsim_results" type="list" label="Muspinsim output collection">
             <discover_datasets pattern="__name__" directory="data" format="txt"/>
         </collection>
-        <data label="muspinsim log for $mu_sim.name" name="log_out" format="txt" from_work_dir="log_out.log"/>
-        <data label="fit report for $mu_sim.name" name="fit_report" format="txt" from_work_dir="fit_report.txt">
+        <data label="muspinsim log on ${on_string}" name="log_out" format="txt" from_work_dir="log_out.log"/>
+        <data label="fit report on ${on_string}" name="fit_report" format="txt" from_work_dir="fit_report.txt">
```
Comment (patrick-austin) [inline muspinsim/muspinsim.xml:55]: Just `muspinsim.xml` missing the wrapper bump.

**Review by patrick-austin — CHANGES_REQUESTED** [review]

File: muspinsim/muspinsim.xml (hunk 1 above)
Comment (subindev-d) [reply muspinsim/muspinsim.xml:55]: fixed it.

**Review by patrick-austin — APPROVED** [review]

File: pm_dftb_opt/pm_dftb_opt.xml
Comment (subindev-d) [inline pm_dftb_opt/pm_dftb_opt.xml:1]: deprecated

**Pull request merged**